reset_user_access_right/models/res_users.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of reset_user_access_right,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     reset_user_access_right is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     reset_user_access_right is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with reset_user_access_right.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, api, exceptions, _
+from openerp.tools import SUPERUSER_ID
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.one
```

Comment:
@adrienpeiffer : "It is strongly recommended to use <b>multi()</b> and either iterate on the self recordset or ensure that the recordset is a single record with <b>ensure_one()</b>"
https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.api.one